docs/guides/Code-Splitting-Using-Holocron.md

```diff
@@ -212,7 +212,7 @@ MyHolocronModule.holocron = {
       const imported = await import(/* webpackChunkName: '<chunkName>' */ './Chunk');
       Chunk = imported.default || imported;
     } catch (error) {
-      Chunk = () => <p>{error.message}</p>;
+      Chunk = function ChunkError() { return <p>{error.message}</p>; };
```

Comment:
why didn't it like the arrow function in a markdown file?

Reply:
it creates an unnamed component